docs/fieldnorm/life_cycle.md

```diff
@@ -1,7 +1,56 @@
 
-Donec mi mi, suscipit venenatis rutrum eu, gravida consectetur justo. Nunc neque ipsum, tincidunt ac libero eu, tincidunt accumsan lorem. Ut aliquet maximus nibh, id volutpat nisi efficitur nec. Nulla et feugiat ipsum. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Curabitur egestas est ac augue hendrerit finibus. Pellentesque hendrerit, magna gravida tincidunt bibendum, velit leo finibus nisi, non sagittis dui purus vitae orci. Etiam lobortis diam sed sem congue lacinia. Phasellus in fermentum libero.
+ Planning
+Planning entails developing a comprehensive software development plan. This includes brainstorming the initial ideas, ideation of the final goal, determining the necessary resources, creating a schedule and establishing achievable milestones. 
+Within a genetics laboratory software development often occurs within the background of laboratory tests. The laboratory therefore needs to develop a solid strategy for change management for its genetic tests that emphasizes all these features, and software development should be an integral part of this strategy. 
+Smaller scale software development and maintenance cycles that might not affect other areas of a genetic lab should maintain a similar strategy on a smaller scale and less laborious, and might use tooling that is more effective for software development, such as code based documentation, an online documentation system that are more suitable for software development planning (like confluence, or wikipages), an issue management system (like issue manager or jira) or combinations thereof.
+Guidelines should cover both strategies, large scale and small scale.
 
-Donec id nibh aliquam, euismod velit id, aliquam ante. Vestibulum finibus interdum tellus quis efficitur. Quisque convallis nec nisl ut sollicitudin. Sed fringilla tellus et massa viverra interdum. Curabitur egestas diam at dui consectetur, et dapibus dui dignissim. Maecenas eget pulvinar enim. Etiam sodales id dui id venenatis. Duis id pellentesque tellus. Quisque enim justo, tristique id volutpat in, euismod ac tellus. Etiam faucibus velit sed purus commodo, eu vehicula velit egestas. Vivamus faucibus est et tortor imperdiet convallis. Vestibulum nulla enim, blandit a purus eget, cursus pellentesque lectus. Nulla finibus mi euismod eros lobortis, vitae eleifend erat rutrum.
\ No newline at end of file
+2.      Requirements
```

Comment:
Trivial suggestion: I think you might do '## Requirements' or '## 2. Requirements' for these headers, the indented spaces now turn your headers into code blocks.